src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.h

```diff
@@ -63,6 +65,7 @@ namespace OpenMS
      * 
      * @param labels    isotopic labels
      * @param missed_cleavages    maximum number of missed cleavages due to incomplete digestion
+     * @param label_mass_shift    name of labels and their corresponding mass shifts
      * For example due to knock-outs in one of the samples.
      */
     MultiplexDeltaMassesGenerator(String labels, int missed_cleavages, std::map<String,double> label_mass_shift);
```

Comment:
The `labels` string is a parameter in `FeatureFinderMultiplex` and `MultiplexResolver`. In order to avoid code duplication, I process the string in this class.